src/server/admin/teams/views/teams-list.njk

```diff
@@ -20,6 +20,7 @@
           { text: "Name", size: "large" },
           { text: "Description", size: "massive" },
           { text: "GitHub team", size: "medium" },
+          { text: "Service Codes", size: "medium" },
```

Comment:
Does this need to be singular?

Reply:
Nope, we're showing the list of service codes here